src/analyses/base.ml

```diff
@@ -26,6 +26,8 @@ module Dep    = BaseDomain.PartDeps
 module WeakUpdates   = BaseDomain.WeakUpdates
 module BaseComponents = BaseDomain.BaseComponents
 
+open Iter
```

Comment:
As far as I can see, this open is unused?

Reply:
Yes, that is correct, this open isn't needed.